CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ For details about compatibility between different releases, see the **Commitment
   - This workaround will be available until the related gateway vendors will release patches for this issue.
 - Firmware version of The Things Kickstarter Gateway are written to the gateway attributes upon receiving a valid status message.
 - Desired mac settings to end device general settings in the Console.
+- Experimental support for Azure Blob Storage. Only authentication via MSI is supported.
```

Comment:
Do people know what MSI is?

Reply:
Well, if they don't then they shouldn't be deploying things on Azure.

https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview

Reply:
The changelog is not a deployment guide